runtime/manual/basics/testing/documentation.md

```diff
@@ -25,13 +32,32 @@ determined by the language identifier attribute which may be any of the
 following:
 
 - `js`
+- `javascript`
+- `mjs`
+- `cjs`
 - `jsx`
 - `ts`
+- `typescript`
+- `mts`
+- `cts`
 - `tsx`
```

Comment:
This list is brought from https://github.com/denoland/deno/blob/8c2f1f5a55a2a9bb9e04c12236faa341b3fd49b6/cli/tools/test/mod.rs#L1183-L1192